### ISSUE TITLE
some semantic error in port_security

### ISSUE THREAD

**Issue by peiqiaoWang** [Emil-501/block.p4#4]
when I run ```p4-validate switch.p4```
some error occured
```
wpq@wpq-S551LN:~/block.p4/port_security/p4src$ p4-validate switch.p4 
parsing successful
Semantic error: Invalid reference to port_sec_metadata in file /home/wpq/block.p4/port_security/p4src/includes/port_security.p4 at line 11: object was not defined
Semantic error: Invalid reference to port_sec_metadata in file /home/wpq/block.p4/port_security/p4src/switch.p4 at line 47: object was not defined
2 errors during semantic checking
Interrupting compilation
```
I check the code ```switch.p4``` and the dirctory ```includes``` follow the cheer, and found that the metadata ```port_sec_metadata``` was not defined.

**Comment by chenxiang2019** [comment]
Hi, @peiqiaoWang 
Thank you for assigning that.

**Comment by chenxiang2019** [comment]
fixed in commit 669cb61
